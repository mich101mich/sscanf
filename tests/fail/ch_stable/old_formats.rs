fn main() {
    sscanf::scanf!("hi", "asdf{x}asdf", usize);
    sscanf::scanf!("hi", "asdf{b}asdf", usize);
    sscanf::scanf!("hi", "asdf{o}asdf", usize);
    sscanf::scanf!("hi", "asdf{r4}asdf", usize);
    sscanf::scanf!("hi", "asdf{r13}asdf", usize);
    sscanf::scanf!("hi", "asdf{/.*/}asdf", usize);
}