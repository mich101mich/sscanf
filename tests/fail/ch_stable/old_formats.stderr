error: It looks like you are using the old format.
       config options now require a ':' prefix like so: {:x}:
       At "asdf{x}asdf"
                ^
 --> $DIR/old_formats.rs:2:26
  |
2 |     sscanf::scanf!("hi", "asdf{x}asdf", usize);
  |                          ^^^^^^^^^^^^^

error: It looks like you are using the old format.
       config options now require a ':' prefix like so: {:b}:
       At "asdf{b}asdf"
                ^
 --> $DIR/old_formats.rs:3:26
  |
3 |     sscanf::scanf!("hi", "asdf{b}asdf", usize);
  |                          ^^^^^^^^^^^^^

error: It looks like you are using the old format.
       config options now require a ':' prefix like so: {:o}:
       At "asdf{o}asdf"
                ^
 --> $DIR/old_formats.rs:4:26
  |
4 |     sscanf::scanf!("hi", "asdf{o}asdf", usize);
  |                          ^^^^^^^^^^^^^

error: It looks like you are using the old format.
       config options now require a ':' prefix like so: {:r4}:
       At "asdf{r4}asdf"
                ^^
 --> $DIR/old_formats.rs:5:26
  |
5 |     sscanf::scanf!("hi", "asdf{r4}asdf", usize);
  |                          ^^^^^^^^^^^^^^

error: It looks like you are using the old format.
       config options now require a ':' prefix like so: {:r13}:
       At "asdf{r13}asdf"
                ^^^
 --> $DIR/old_formats.rs:6:26
  |
6 |     sscanf::scanf!("hi", "asdf{r13}asdf", usize);
  |                          ^^^^^^^^^^^^^^^

error: It looks like you are using the old format.
       config options now require a ':' prefix like so: {:/.*/}:
       At "asdf{/.*/}asdf"
                ^^^^
 --> $DIR/old_formats.rs:7:26
  |
7 |     sscanf::scanf!("hi", "asdf{/.*/}asdf", usize);
  |                          ^^^^^^^^^^^^^^^^
